extensions/redis-client/runtime/src/main/java/io/quarkus/redis/runtime/client/VertxRedisClientFactory.java

```diff
@@ -99,6 +101,21 @@ public static Redis create(String name, Vertx vertx, RedisClientConfig config) {
         return Redis.createClient(vertx, options);
     }
 
+    private static URI addClientParameter(String clientName, URI uri) {
+        try {
+            String query = uri.getQuery() == null ? "client=%s".formatted(clientName)
+                    : uri.getQuery().formatted("&client=%s", clientName);
+            return new URI(
+                    uri.getScheme(),
+                    uri.getAuthority(),
+                    uri.getPath(),
+                    query,
+                    uri.getFragment());
+        } catch (URISyntaxException e) {
+            return uri;
```

Comment:
Not sure if we need to log and return or fail because it means the client name could not have been applied.

Reply:
Logging a warning is probably a good idea, but failing just because of the client name seems a bit excessive.